backend/game/src/game_options.rs

```diff
@@ -49,11 +50,23 @@ pub struct GameOptions {
     action_cards: HashMap<ActionCard, ActionCardInfo>,
     /// What agendas exist in the game.
     agendas: HashMap<Agenda, AgendaInfo>,
+    /// What leaders exist in the game.
+    leaders: HashMap<Leader, LeaderInfo>,
+    /// Map from all factions in the game to the leaders of that faction.
```

Comment:
Denna känns ganska onödig IMO, iaf för frontenden. Vi har ju inte motsvarande för tech eller liknande så känns lite inkonsekvent också

Reply:
Slängde in den som ett test för att det kanske skulle göra saker lättare i frontenden. Om den inte hjälper så tar vi bort den.